debugger/src/main/kotlin/org/rust/debugger/lang/RsMSVCTypeNameDecoratorVisitor.kt

```diff
@@ -0,0 +1,186 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.lang
+
+import org.rust.debugger.lang.RustTypeNameParser.*
+
+class RsMSVCTypeNameDecoratorVisitor : RustTypeNameBaseVisitor<Unit>() {
```

Comment:
BTW maybe use `Rs` prefix for generated code as well instead if `Rust`?
To be consistent in all codebase

Reply:
Are you sure? It seems it still has `Rust` prefix